sources/Debug.class.php

```diff
@@ -181,14 +181,15 @@ public function get_db()
 
 	/**
 	 * Adds a new getrusage value (by default two are added: one at the beginning
-	 * of the script execution and one at the end
+	 * of the script execution and one at the end)
 	 *
 	 * @param string $point can be end or start depending on when the function
 	 *               is called
 	 * @param string[]|null $rusage value of getrusage or null to let the method call it
 	 */
 	public function rusage($point, $rusage = null)
 	{
+		// getrusage is missing in php < 7 on Windows
 		if (!function_exists('getrusage') || !$this->_track)
```

Comment:
NitPik I'd reverse this order .. no need to do the function exists if you are not tracking to begin with.